bin/run

```diff
@@ -74,7 +74,7 @@ init_connections() {
         --conn_id databricks_default \
         --conn_type databricks \
         --conn_host https://dbc-caf9527b-e073.cloud.databricks.com \
-        --conn_extra "{\"token\":\"${DB_TOKEN}\"}"
+        --conn_extra "{\"token\":\"${DB_TOKEN}\", \"host\": \"\"}"
```

Comment:
some databricks operator change

Reply:
Appears to be the case according to https://github.com/apache/airflow/pull/5635/files#diff-53a81d33b2423e1a6b28c14b7e5642d1